index.d.ts

```diff
@@ -0,0 +1,209 @@
+declare module 'whatwg-fetch' {
+  import {Agent} from 'http'
+  import {URLSearchParams, URL} from 'url'
```

Comment:
Once the node-fetch extensions get dropped, I bet you can delete these imports.  You should only need native JS types to create this declaration file.

Reply:
Done.